metrics/prometheus-agent/values.yaml

```diff
@@ -62,3 +66,27 @@ prometheus:
     externalLabels: {}
   serviceAccount:
     create: true
+  serviceMonitor:
+    interval: ""
+    selfMonitor: true
+
+    sampleLimit: 0
+
+    targetLimit: 0
+
+    labelLimit: 0
+
+    labelNameLengthLimit: 0
+
+    labelValueLengthLimit: 0
+
+    scheme: ""
+
+    tlsConfig: {}
+
+    bearerTokenFile:
+
+    metricRelabelings: []
+    
+    relabelings: []
```

Comment:
since these whole configs are related to servicemonitor , i would remove the extra lines between so it wont be confusing and be clear its all servicemonitor configs

Reply:
Done :D